src/main/java/catalin/Writer.java

```diff
@@ -0,0 +1,49 @@
+package catalin;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Writer {
+
+    public static void writeMatrix(File fileName, Matrix matrix) {
+        BufferedWriter bufferedWriter = null;
+        FileWriter fileWriter = null;
+
+        try {
+
+
+            fileWriter = new FileWriter(fileName.getAbsoluteFile(), true);
+            bufferedWriter = new BufferedWriter(fileWriter);
```

Comment:
You can take advantage and use _try-with-resources_ .
you can replace and user directly try (FileWriter bufferedWriter = new FileWriter(fileName.getAbsoluteFile(), true);)